app_builder/lib/app_builder/macos.ex

```diff
@@ -142,7 +142,7 @@ defmodule AppBuilder.MacOS do
     #!/bin/sh
     set -e
     root=$(dirname $(dirname "$0"))
-    $root/Resources/rel/bin/#{release_name} start \
+    LIVEBOOK_ROOT_PATH=$HOME $root/Resources/rel/bin/#{release_name} start \
```

Comment:
we shouldn't put Livebook specific stuff into `app_builder/`.

Instead, let's set this env var in `rel/app/env.*` and we should be good to go!

Reply:
@wojtekmach Yes, that makes sense.
I just updated the PR.